src/pcrlock/pcrlock.c

```diff
@@ -2024,6 +2024,7 @@ static int add_algorithm_columns(
 
 static int show_log_table(EventLog *el, JsonVariant **ret_variant) {
         _cleanup_(table_unrefp) Table *table = NULL;
+        uint32_t pcr_mask = arg_pcr_mask ?: TPM2_PCRS_MASK;
```

Comment:
hmm, we try to avoid relying on C's downgrade-to-bool feature for numeric values that are not actually boolens or boolean-like. Hence, please spell this out as `… arg_pcr_mask != 0 ? arg_pcr_mask : TPM2_PCRS_MASK`, event if entirely equivalent.